.gitignore

```diff
@@ -1 +1,33 @@
-addon.xml
\ No newline at end of file
+# build artifacts
+build/
+obj-x86_64-linux-gnu/
+game.*/addon.xml
+
+# commonly used editors
+# vim
+*.swp
+
+# Eclipse
+*.project
+*.cproject
+.classpath
+*.sublime-*
+.settings/
+
+# KDevelop 4
+*.kdev4
+
+# gedit
+*~
+
+# CLion
+/.idea
+
+# clion
+.idea/
+
+# KDev
+*.kdev4
+
+*.patch
```

Comment:
Remove these two lines, .patch files placed in depends/common/2048 are applied to the 2048 source